init/module-setup.sh

```diff
@@ -25,6 +25,11 @@ kdump_check_net() {
     elif [ "${KDUMP_NETCONFIG%:force}" != "$KDUMP_NETCONFIG" ]; then
         # always set up network
         kdump_neednet=y
+    elif [ -f "/usr/lib/fence_kdump_send" ] &&
```

Comment:
Better use a global variable, so if the location of the script changes, it can be adjusted in one place only, e.g. FENCE_KDUMP_SEND=/usr/lib/fence_kdump_send

Reply:
Fixed in the new commit with the global variable.